.github/workflows/main.yml

```diff
@@ -0,0 +1,18 @@
+name: build
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Package and run all tests
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - name: Run Maven Targets
+      run: mvn package
```

Comment:
You should be able to run all the targets with this build - checkstyle, jacoco, spotbugs, coveralls